Modules/xASL_module_ASL.m

```diff
@@ -246,8 +246,7 @@
 
 if x.modules.asl.bMergingSessions
     fprintf('%s', ' and will now concatenate the following sessions:')
-    nSessions = numel(x.modules.asl.sessionsToMerge);
-    for iSession=1:nSessions
+    for iSession = 1:numel(x.modules.asl.sessionsToMerge)
```

Comment:
I thought we put nSessions here to reuse it also later

Reply:
Yes. That was the entire point of my edit.
nSessions was only used here on this single occasion. And once you define such a variable, it increases the risk of a bug later. Because then somebody will see `nSession` and will reuse it without really knowing if that's number of sessions for this subject, or a number of sessions in a merging list etc. So this highly increases the risk of incorrect use later on through negligence.

Reply:
good point